leapp/snactor/commands/workflow/sanity_check.py

```diff
@@ -24,10 +28,19 @@ def print_fail(error):
     print('{red}FAIL: {error}{reset}'.format(red=Color.red, error=error, reset=Color.reset), file=sys.stderr, end='\n')
 
 
+def _ignore_file_content(path):
+    if path:
+        with open(path) as f:
+            return [line.strip() for line in f.read().split('\n') if not line.strip().startswith('#')]
```

Comment:
@vinzenz one small fix to prevent crashes when no ignorefile is specified:
```suggestion
    if not path:
        return []
    with open(path) as f:
        return [line.strip() for line in f.read().split('\n') if not line.strip().startswith('#')]
```

before:
```
Traceback (most recent call last):                                              
  File "/repocopy/tut/bin/snactor", line 8, in <module>                            
    sys.exit(main())                                                               
  File "/repocopy/tut/lib/python3.6/site-packages/leapp/snactor/__init__.py", line 78, in main
    cli.command.execute(version=_('snactor version {}').format(VERSION))           
  File "/repocopy/tut/lib/python3.6/site-packages/leapp/utils/clicmd.py", line 111, in execute
    args.func(args)                                                                
  File "/repocopy/tut/lib/python3.6/site-packages/leapp/utils/clicmd.py", line 133, in called
    self.target(args)                                                              
  File "/repocopy/tut/lib/python3.6/site-packages/leapp/utils/repository.py", line 20, in checker
    return f(*args, **kwargs)                                                      
  File "/repocopy/tut/lib/python3.6/site-packages/leapp/snactor/commands/workflow/sanity_check.py", line 43, in cli 
    ignored_models = set(params.ignore or []) | set(_ignore_file_content(params.ignorefile))

```